pages/aboutPage.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="h-full">
+        <TopNav />
+        <div class="flex flex-col md:flex-row h-full">
+            About us!
+
+            <a href="https://ko-fi.com/Z8Z14YNUR" target="_blank">
+                <img height="36" style="border: 0px; height: 36px" src="https://cdn.ko-fi.com/cdn/kofi1.png?v=3"
+                    alt="Buy Me a Coffee at ko-fi.com" />
+            </a>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+export default defineComponent({
+})
+</script>
```

Comment:
```suggestion

```

Reply:
Is there a difference in the suggestion? 😅  I can't tell

Reply:
ohhhh...it's to remove it....I'm literally re-adding them in in future commits. For now, we should leave them. Especially considering I'm the sole developer on these for now. Before we have other contributors work on them, the files will be completely different